Core/Security/VOMSService.py

```diff
@@ -105,3 +131,41 @@ def attGetUserNickname( self, dn, ca ):
     else:
       return S_ERROR( result )
 
+  def getUsers( self ):
+    """ Get all the users of the VOMS VO with their detailed information
+
+    :return: user dictionary keyed by the user DN
+    """
+
+    vomsUsers = {}
+
+    result = self.admListMembers()
+    if not result['OK']:
+      return result
+    members = result['Value']
+
+    result = self.admListRoles()
+    if not result['OK']:
+      return result
+    roles = result['Value']
+
+    roleMembers = {}
+    for role in roles:
+      result = self.admListUsersWithRole( '/%s' % self.vomsVO, role )
```

Comment:
double checking: does role starts with "Role=", like in https://github.com/DIRACGrid/DIRAC/blob/integration/ConfigurationSystem/Agent/UsersAndGroups.py#L262 ? Also, at same location, the first parameter to admListUsersWithRole is passed in without "/"